### PR TITLE
update alarms collector readme image to use one from the related netdata PR

### PR DESCRIPTION
**Pull request by andrewm4894**
update readme image to use one from the related netdata PR as opposed to coming from a private repo.

##### Component Name

docs

### CODE REVIEW

**Review by joelhans — APPROVED** [review]

File: collectors/python.d.plugin/alarms/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This collector creates an 'Alarms' menu with one line plot showing alarm states
 
 Below is an example of the chart produced when running `stress-ng --all 2` for a few minutes. You can see the various warning and critical alarms raised. 
 
-![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-alarms-collector.jpg)
+![alarms collector](https://user-images.githubusercontent.com/2178292/101505477-4f9ef200-396c-11eb-864c-a44d9eaf2ee5.png)
```
Comment (ilyam8) [inline collectors/python.d.plugin/alarms/README.md:22]: as i mentioned https://github.com/netdata/netdata/pull/10042#issuecomment-740702959

this image is not up to date:
 - the title has not status mapping
 - wrong icon (❗ => 🔔 )
 - wrong priority - the chart below `Netdata Monitoring`

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: collectors/python.d.plugin/alarms/README.md (hunk 1 above)
Comment (joelhans) [inline collectors/python.d.plugin/alarms/README.md:22]:
```suggestion
![alarms collector](https://user-images.githubusercontent.com/1153921/101641493-0b086a80-39ef-11eb-9f55-0713e5dfb19f.png)
```

**Review by joelhans — CHANGES_REQUESTED** [review]
Just took a new screenshot with some RAM alarms and popped it into a suggestion. I think it resolves the concerns from @ilyam8.

File: collectors/python.d.plugin/alarms/README.md (hunk 1 above)
Comment (andrewm4894) [reply collectors/python.d.plugin/alarms/README.md]: This looks to be fixed following the suggested update from @joelhans

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Review by joelhans — APPROVED** [review]

**Pull request merged**